core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -157,10 +158,10 @@ func saveReplacementInProgressAttempt(store *strpkg.Store, oldAttempt models.Eth
 		return errors.New("expected oldAttempt to have an ID")
 	}
 	return store.Transaction(func(tx *gorm.DB) error {
-		if err := tx.Exec(`DELETE FROM eth_tx_attempts WHERE id = ? `, oldAttempt.ID).Error; err != nil {
+		if err := tx.Omit(clause.Associations).Exec(`DELETE FROM eth_tx_attempts WHERE id = ? `, oldAttempt.ID).Error; err != nil {
```

Comment:
Does this really make a difference? I would have thought `Exec` doesn't trigger any callbacks related to associations.

Reply:
Ah, this is superfluous. Left it there by accident.

I assume you will be removing them all when you purge associations permanenty?

Reply:
Yes I will be.